### DISCUSSION TITLE
[Helm] Support the GKE Workload Identity

### DISCUSSION THREAD

**Discussion by chrisduong** [airbytehq/airbyte#44884] [Deployment]
It had been mentioned here https://discuss.airbyte.io/t/logging-to-gcs-with-workload-identity-in-airbyte-on-gke/6731

In the security point of view, we should use the GKE Workload Identity instead. This means we don't need to set `global.storage.gcs.credentialsJson`.

However it is not possible, as the Helm chart always requires it.

The remedy is to update the Helm chart to allow `global.storage.gcs.credentialsJson` to be empty.

**Comment by NAjustin** [comment]
It's worth noting that there's already a (very stale) issue here for this:
https://github.com/airbytehq/airbyte/issues/13081

I agree this would be the ideal solution though when deploying on GKE.

[a comment by an automated account is not shown]

**Comment by MorEvinced** [comment]
Hi, @airbyte-support-bot , This is still relevant - can you re-open please? I can't use airbyte safetly via WI until this is fixed
